### PR TITLE
Self test 07.20.2019 - Task was completely changed

### PR DESCRIPTION
**Pull request by novikovkiruha**

### CODE REVIEW

**Review by alekseybeletsky — APPROVED** [review]
Good job!

**Review by alekseybeletsky — APPROVED** [review]
Please don't forget to unsubscribe from events.

File: self_test_07.20.2019/FridgeStatusEventArgs.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        public DoorStatus FreezerDoor { get; set; }
+
+        public string Message { get; }
+
+        public FridgeStatusEventArgs(string message)
+        {
+            this.Message = message;
+        }
+
+        public FridgeStatusEventArgs(SwitchStatus fridgeStatus, DoorStatus fridgeDoor, DoorStatus freezerDoor)
```
Comment (alekseybeletsky) [inline self_test_07.20.2019/FridgeStatusEventArgs.cs]: This constructor is not used.

File: self_test_07.20.2019/Fridge.cs [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                        Console.ForegroundColor = ConsoleColor.Red;
+                        break;
+                    case "0":
+                        break;
+                    default:
+                        Console.WriteLine("Wrong option. Try again...");
+                        continue;
+                }
+
+                this.OnNewFridge(new FridgeStatusEventArgs($"Fridge door position: {this.FridgeDoor}{Environment.NewLine}"));
```
Comment (alekseybeletsky) [inline self_test_07.20.2019/Fridge.cs]: According to the task description, we should also notify about current state.

Comment (alekseybeletsky) [reply self_test_07.20.2019/Fridge.cs]: -2 to logic.

File: self_test_07.20.2019/FridgeStatusEventArgs.cs (hunk 1 above)
Comment (novikovkiruha) [reply self_test_07.20.2019/FridgeStatusEventArgs.cs]: I know, but at first I was going to use it. Constructor was deleted. Fixed.

File: self_test_07.20.2019/Fridge.cs (hunk 2 above)
Comment (novikovkiruha) [reply self_test_07.20.2019/Fridge.cs]: fixed

**Pull request closed without merging**